app/src/main/java/com/aol/mobile/sdk/testapp/tutorials/three/LoopPlaybackActivity.java

```diff
@@ -32,8 +33,13 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
             @Override
             public void onPlayerStateChanged(@NonNull Properties properties) {
                 VideoProperties videoProperties = properties.playlistItem.video;
-                if (videoProperties != null && videoProperties.time != null && videoProperties.time.progress == 1) {
-                    playerFragment.getBinder().getPlayer().replay();
+                if (properties.playlist.isLastVideo && videoProperties != null && videoProperties.time != null && videoProperties.time.progress == 1) {
+                    new Handler().post(new Runnable() {
```

Comment:
Each time new `Handler` when state is updated?

Reply:
Fixed to reuse only one